internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -85,12 +85,24 @@ func resourceDatabricksWorkspace() *pluginsdk.Resource {
 			},
 
 			"managed_services_cmk_key_vault_key_id": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ValidateFunc: keyVaultValidate.KeyVaultChildID,
+			},
+
+			"managed_disk_cmk_key_vault_key_id": {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ForceNew:     true,
 				ValidateFunc: keyVaultValidate.KeyVaultChildID,
 			},
 
+			"managed_disk_cmk_rotation_to_latest_version_enabled": {
+				Type:         pluginsdk.TypeBool,
+				Optional:     true,
+				RequiredWith: []string{"managed_disk_cmk_key_vault_key_id"},
```

Comment:
It's not required. This flag only ensures that if this argument is set, the former one must be set as well, so that people don't set this and expect it to implicitly work.